src/core/thread/mle_constants.hpp

```diff
@@ -126,7 +126,7 @@ enum
 enum
 {
     kMplChildDataMessageTimerExpirations  = 0, ///< Number of MPL retransmissions for Children.
-    kMplRouterDataMessageTimerExpirations = 2, ///< Number of MPL retransmissions for Routers.
+    kMplRouterDataMessageTimerExpirations = 1, ///< Number of MPL retransmissions for Routers.
```

Comment:
It violates Thread Specification. If it works badly, maybe we should point it out in the Thread Group.